lib/views/pages/login_signup/login_form.dart

```diff
@@ -64,20 +61,24 @@ class LoginFormState extends State<LoginForm> {
       setState(() {
         _progressBarState = false;
       });
-      _exceptionToast(
-          'Connection Error. Make sure your Internet connection is stable');
+      Fluttertoast.showToast(
+          msg: 'Connection Error. Make sure your Internet connection is stable',
+          backgroundColor: UIData.toastErrorColor);
     } else if (result.hasException) {
       print(result.exception);
       setState(() {
         _progressBarState = false;
       });
 
-      _exceptionToast(result.exception.toString().substring(16, 35));
+      Fluttertoast.showToast(
```

Comment:
Hi @utkarshshendge 
Can you break the exception here like whether it is `GraphqlError` or it is `ClientException` and display message accordingly. It would enhance the experience of user as he/she gets to know whether it is a server side problem or something related to their phone connectivity.

Reply:
@sumitra19jha this is much needed. But it looks a bit out of the scope of this PR.
It will be best if a new issue is created for it so we can apply these changes in the entire App

Reply:
@utkarshshendge if possible add the changes as part of this PR only. Update issue and PR description. 
Also fix conflicts on this PR.